lib/replaceWithDecompressedPrimitive.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+var Cesium = require('cesium');
+var draco3d = require('draco3d');
+
+module.exports = replaceWithDecompressedPrimitive;
+
+/**
+ * Replace the accessor properties of the original primitive with the values from the decompressed primitive.
+ *
+ * @param {Object} gltf A javascript object containing a glTF asset.
+ * @param {Object} primitive A javascript object containing a glTF primitive.
+ * @param {Number} encodedPointsCount Number of points after the mesh is decompressed.
+ * @param {Number} encodedFacesCount Number of faces after the mesh is decompressed.
+ * @param {Object} dracoExtension A javascript object containing KHR_draco_mesh_compression.
+ * @returns {Object} The glTF asset with the decompressed primitive.
+ *
+ * @private
+ */
+function replaceWithDecompressedPrimitive(gltf, primitive, encodedPointsCount, encodedFacesCount, dracoExtension) {
```

Comment:
Now that this function is simpler I think it's fine to move it into `compressDracoMeshes`.

Reply:
I'm going to keep for now, as the next branch will add more code here.

Reply:
Alright!